docu/release_notes/2016-10-28-Technev150.html.md

```diff
@@ -0,0 +1,21 @@
+---
+date: October 28, 2016
+service: 'hybris techne'
+official_version: v1.5.0
+headline: 'Techne v1.5.0 released, New Product Thumbnail component. Revisited Tree pattern.'
+---
+
+<h1>Techne v1.5.0</h1>
+<p>New Product Thumbnail pattern added. Can be used to show a list of products in a grid.</p>
+<p>New Grid Started pages added. Speeds the development of pages used to display products in a grid. It has also our new toolbar for easy filtering and navigation.</p>
+<p>Revisited Tree pattern, along with respective Started page. Now with more standard expand/collapse icons and behavior. Description of how the interactivity should work.</b>
+
```

Comment:
Let's break this down into two bullet items one for the pattern and another for the starer page 

- Updated Tree pattern. Now with more standard expand/collapse icons and behavior and description of how the interactivity should work.
- New Tree Starter Page added to show the interactive behaviour and rules in action